src/router.js

```diff
@@ -80,15 +80,14 @@ export class Router {
   * @param viewPort The viewPort.
   * @param name The name of the viewPort. 'default' if unspecified.
   */
-  registerViewPort(viewPort:Object, name?:string) {
-    name = name || 'default';
+  registerViewPort(viewPort : Object, name? : string = 'default') : void {
```

Comment:
It doesn't break it, because `default` is assumed only when `undefined` is passed. You can check it out. 

```javascript
function defaults(x = 'default') {
  console.log(x);
}

defaults(''); -> prints ''
defaults('view'); -> prints 'view'
defaults(); -> prints 'default'
```
Check here: http://www.es6fiddle.net/idjr4tkv/